packages/test-renderer/package.json

```diff
@@ -24,6 +24,7 @@
   "peerDependencies": {
     "react": ">=17.0",
     "@react-three/fiber": ">=7.0.7",
+    "@react-three/shared": ">=7.0.7",
```

Comment:
@joshuaellis Should @react-three/shared be a dependency or a peerDependency of test-renderer?